system-contracts/contracts/libraries/SystemContractHelper.sol

```diff
@@ -224,9 +224,9 @@ library SystemContractHelper {
     /// that a single byte sent to L1 as pubdata costs.
```

Comment:
Given the changes made in the `return` field of the natspec comments, the `@notice` field looks incorrect to me.